src/BuildScriptGeneratorCli/Commands/BuildCommand.cs

```diff
@@ -121,21 +120,6 @@ internal override int Execute(IServiceProvider serviceProvider, IConsole console
 
             console.WriteLine(buildInfo.ToString());
 
-            // Try writing the ID to a file in the source directory
```

Comment:
Just FYI...I am making this change because the file containing the operation id was being put in the source directory and not the destination directory. This breaks scenario where the source directory is not the same as the destination directory (ex: dotnetcore and also if a different directory is provided as output for node, python or php). In this PR we store this information in the manifest file which the runtime startup scripts read to get the operation id.

I have tested this E2E (using our test AI instance) for all platforms and the correlation happens end-to-end.

Reply:
We could have just moved the build ID file to the destination directory, no?

Reply:
Yes, that idea crossed my mind, but we already have a way to describe a built state with the manifest file so we need not have another one specifically for build id